xml/operations-maintenance-controller-pit_swiftrings_recovery.xml

```diff
@@ -6,110 +6,163 @@
+Jun 18 20:01:49 ardana-qe102-cp1-c1-m1-mgmt swiftlm-uptime-mon[3985]: 'uptime-mon - INFO : Metric:keystone-get-token:avg-latency: 0.679254770279 (at 1529352109.66)'
+
+PLAY RECAP ********************************************************************
+ardana-qe102-cp1-c1-m1 : ok=12 changed=0 unreachable=0 failed=0```</screen>
+     </step>
+     <step>
+      <para>
+       Find the first node name and its IP address. For example:
+      </para>
+<screen>cat /etc/hosts | grep ardana-qe102-cp1-c1-m1</screen>
+     </step>
+    </substeps>
+   </step>
+   <step>
     <para>
      Become root:
     </para>
 <screen>sudo su</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Source the backup environment file:
     </para>
-<screen>source /home/stack/backup.osrc</screen>
-   </listitem>
-   <listitem>
+<screen>source /var/lib/ardana/backup.osrc</screen>
+   </step>
+   <step>
+    <para>
+     Find the client id.
+    </para>
+<screen>freezer client-list
+<?dbsuse-fo font-size="0.65em"?>
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| Client ID                   | uuid                             | hostname                    | description |
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| ardana002-cp1-comp0001-mgmt | f4d9cfe0725145fb91aaf95c80831dd6 | ardana002-cp1-comp0001-mgmt |             |
+| ardana002-cp1-comp0002-mgmt | 55c93eb7d609467a8287f175a2275219 | ardana002-cp1-comp0002-mgmt |             |
+| ardana002-cp1-c0-m1-mgmt    | 50d26318e81a408e97d1b6639b9404b2 | ardana002-cp1-c0-m1-mgmt    |             |
+| ardana002-cp1-c1-m1-mgmt    | 78fe921473914bf6a802ad360c09d35b | ardana002-cp1-c1-m1-mgmt    |             |
+| ardana002-cp1-c1-m2-mgmt    | b2e9a4305c4b4272acf044e3f89d327f | ardana002-cp1-c1-m2-mgmt    |             |
+| ardana002-cp1-c1-m3-mgmt    | a3ceb80b8212425687dd11a92c8bc48e | ardana002-cp1-c1-m3-mgmt    |             |
++-----------------------------+----------------------------------+-----------------------------+-------------+</screen>
+    <para>
+     In this example, the client id of host
+     <literal>ardana002-cp1-c1-m1-mgmt</literal> is
+     <literal>ardana002-cp1-c1-m1-mgmt</literal>.
+    </para>
+   </step>
+   <step>
+    <para>
+     List the jobs
+    </para>
+<screen>freezer job-list -C <replaceable>HOSTNAME</replaceable></screen>
     <para>
-     List the jobs:
+     Using the example in the previous step:
     </para>
-<screen>freezer-scheduler -c &lt;hostname&gt; job-list</screen>
-   </listitem>
-   <listitem>
+<screen>freezer job-list -C ardana002-cp1-c1-m1-mgmt</screen>
+   </step>
+   <step>
     <para>
-     Get the id corresponding to the job <literal>&lcm; Default: Swift restore
-     from Swift</literal>.
+     Get the corresponding job id for <literal>&lcm; Default: &swift; restore
+     from &swift;</literal>.
```

Comment:
This sentence does not make sense to me. Is "&lcm; Default: Swift restore from Swift" a column from the output of the previous command? Then we should name it as such (Get the job ID from the column / Get the job ID from the line starting with", and this text should be part of the previous step.

Reply:
see previous comments